braindecode/datautil/transforms.py

```diff
@@ -0,0 +1,115 @@
+"""Transforms that work on Raw or Epochs objects.
+ToDo: should transformer also transform y (e.g. cutting continuous labelled 
+      data)?
+"""
+
+# Authors: Hubert Banville <hubert.jbanville@gmail.com>
+#          Lukas Gemein <l.gemein@gmail.com>
+#          Simon Brandt <simonbrandt@protonmail.com>
+#          David Sabbagh <dav.sabbagh@gmail.com>
+#
+# License: BSD (3-clause)
+
+import mne
+
+
+class FilterRaw(object):
```

Comment:
I would try to have only one Filter class that is smart enough to call .filter if method is there or just work with ndarray.

Reply:
the only thing is that if you have windows you need to pass explicitly the sfreq which might deserve a discussion